backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepository.java

```diff
@@ -30,49 +32,124 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(DemoOktaRepository.class);
 
+    DemoUserConfiguration demoUsers;
     Map<String,OrganizationRoleClaims> usernameOrgRolesMap;
     Map<String,Map<OrganizationRole,List<String>>> orgRoleUsernamesMap;
+    Set<String> inactiveUsernames;
 
     public DemoOktaRepository(DemoUserConfiguration demoUsers) {
+        this.demoUsers = demoUsers;
         this.usernameOrgRolesMap = new HashMap<>();
         this.orgRoleUsernamesMap = new HashMap<>();
+        this.inactiveUsernames = new HashSet<>();
 
-        for (DemoUser altUser : demoUsers.getAlternateUsers()) {
-            IdentityAttributes user = altUser.getIdentity();
-            String username = user.getUsername();
-            String orgExternalId = altUser.getAuthorization().getOrganizationExternalId();
-            Set<OrganizationRole> roles = altUser.getAuthorization().getGrantedRoles();
-            OrganizationRoleClaims orgRoles = new OrganizationRoleClaims(orgExternalId, roles);
-
-            usernameOrgRolesMap.put(username, orgRoles);
-
-            Map<OrganizationRole,List<String>> roleUsernamesMap = 
-                    orgRoleUsernamesMap.getOrDefault(orgExternalId, new HashMap<>());
-            for (OrganizationRole role : roles) {
-                List<String> usernames = roleUsernamesMap.getOrDefault(role, new ArrayList<>());
-                usernames.add(username);
-                roleUsernamesMap.put(role, usernames);
-            }
+        reset();
 
-            orgRoleUsernamesMap.put(orgExternalId, roleUsernamesMap);
+        LOG.info("Done initializing Demo Okta service.");
+    }
 
+    private void initDemoUser(DemoUser demoUser) {
+        IdentityAttributes user = demoUser.getIdentity();
+        String username = user.getUsername();
+        String orgExternalId = demoUser.getAuthorization().getOrganizationExternalId();
+        Set<OrganizationRole> roles = new HashSet<>();
+        roles.add(OrganizationRole.USER);
```

Comment:
I think the initial demo user should be an admin

Reply:
they already are, actually! it's specified in `application-no-security.yaml` and picked up here through `demoUser.getAuthorization().getGrantedRoles()`